pilot/pkg/security/authn/v1beta1/policy_applier.go

```diff
@@ -59,10 +63,68 @@ func (a *v1beta1PolicyApplier) JwtFilter(isXDSMarshalingToAnyEnabled bool) *http
 	return out
 }
 
+// All explaining code link can be removed before merging.
+func convertToIstioAuthnFilterConfig(jwtRules []*v1beta1.JWT) *authn_filter.FilterConfig {
+	p := authn_alpha.Policy{
+		// Targets are not referenced in proxy repo.
+		// https://github.com/istio/istio/blob/d8eb9d3699ef8945aa4cc6a1001a5455683df3b5/pilot/pkg/security/authn/v1alpha1/policy_applier.go#L265
+
+		// Origin will be optional since we don't reject req.
+		// Add Peers since we need that to trigger identity extraction in Authn filter.
+		// https://github.com/istio/proxy/blob/24e971ff31c5cce22e9ab49f8478629f50664846/src/envoy/http/authn/peer_authenticator.cc#L41
+		Peers: []*authn_alpha.PeerAuthenticationMethod{
+			&authn_alpha.PeerAuthenticationMethod{
+				Params: &authn_alpha.PeerAuthenticationMethod_Mtls{
+					Mtls: &authn_alpha.MutualTls{},
+				},
+			},
+		},
+		// https://github.com/istio/proxy/search?q=targets&unscoped_q=targets
+		OriginIsOptional: true,
+		// Peers not really needed, always invoke the
+		// https://github.com/istio/proxy/blob/24e971ff31c5cce22e9ab49f8478629f50664846/src/envoy/http/authn/http_filter.cc#L84
```

Comment:
Here and other places, I feel it's uncommon to add comments that link to other repo like this. It looks duplicate and also is hard to maintain.

Reply:
Yeah that'd why I have a comment above func to remove it before merging